main.go

```diff
@@ -239,6 +239,7 @@ func getReadyNodesAndNumberOfNonReadyNodesOrInstances(updatedInstances []*autosc
 		} else if kubeletCondition := conditions[len(conditions)-1]; kubeletCondition.Type == v1.NodeReady && kubeletCondition.Status == v1.ConditionTrue {
 			updatedReadyNodes = append(updatedReadyNodes, updatedNode)
 		} else {
+			log.Printf("[%s][%s] Skipping because kubelet on node is not reporting as ready", aws.StringValue(autoScalingGroup.AutoScalingGroupName), aws.StringValue(updatedInstance.InstanceId))
```

Comment:
Shouldn't we log the actual reported condition for verbosity instead of just assuming that the `Ready` condition is `True`?

It could be `Unknown`, or the condition type could (somehow) not be `Ready`.

I'd rather have something like this:
```go
} else if kubeletCondition := conditions[len(conditions)-1]; kubeletCondition.Type == v1.NodeReady {
    if kubeletCondition.Status == v1.ConditionTrue {
        updatedReadyNodes = append(updatedReadyNodes, updatedNode)
    } else {
        log.Printf("[%s][%s] Skipping because kubelet condition %s is reporting as %s", aws.StringValue(autoScalingGroup.AutoScalingGroupName), aws.StringValue(updatedInstance.InstanceId), kubeletCondition.Type, kubeletCondition.Status)
        numberOfNonReadyNodesOrInstances++
    }
} else {
    log.Printf("[%s][%s] Skipping because expected kubelet on node to have condition %s with value %s, but it didn't", aws.StringValue(autoScalingGroup.AutoScalingGroupName), aws.StringValue(updatedInstance.InstanceId), v1.NodeReady, v1.ConditionTrue)
    numberOfNonReadyNodesOrInstances++
```
I don't think the code in the else block should ever be reached, but better safe than sorry 🤷